test/v2/connectors/CreateForeignPropertyPeriodSummaryConnectorSpec.scala

```diff
@@ -159,7 +159,7 @@ class CreateForeignPropertyPeriodSummaryConnectorSpec extends ConnectorSpec {
 
```

Comment:
There really is no need to separately test with consolidated expenses. The connector spec doesn't care what the body of the submission is (the mocked http client doesn't even convert it to JSON) so the body can be as simple an example as you like.